evm-contracts/src/v0.5/dev/PrepaidAggregator.sol

```diff
@@ -114,6 +119,8 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     onlyValidRoundId(uint32(_round))
     onlyValidOracleRound(uint32(_round))
   {
+    emit SubmissionReceived(_answer, uint32(_round), msg.sender);
```

Comment:
We should emit this within recordSubmission to follow the convention set in this contract (emitting within the private functions).